modules/courses/public/views/pages/api/admin/users.csv.liquid

```diff
@@ -0,0 +1,24 @@
+{%- liquid
+  function current_user = 'modules/user/lib/queries/user/current'
+  function can = 'modules/permission/lib/helpers/can_do', requester: current_user, do: 'users.manage'
+  unless can
+    redirect_to '/course'
+  endunless
+
+  graphql profiles = 'modules/courses/profile/search', per_page: 2000
```

Comment:
this probably will not work due to timeout, the way to go would be to create a background job and upload the result file to s3 ; best practice would be to encrypt the content of the file before uploading it